docs/configuration/network_filters/client_ssl_auth_filter.rst

```diff
@@ -13,7 +13,7 @@ Client TLS authentication filter :ref:`architecture overview <arch_overview_ssl_
     "config": {
       "auth_api_cluster": "...",
       "stat_prefix": "...",
-      "refresh_interval_ms": "...",
+      "refresh_delay_ms": "...",
```

Comment:
I just realized that CLIENT_SSL_NETWORK_FILTER_SCHEMA doesn't have `refresh_delay_ms`. Can you please add that?